contacts.py

```diff
@@ -121,7 +128,11 @@ def _load(self):
                     relative_file_path = FSBackedThreeLevelDict.get_file_path_from_file_name(file_name)
                     # Note this is expensive, it has to read each file to find update_tokens
                     # - maintaining an index would be better.
-                    blob = json.load(open('/'.join([root, file_name])))
+                    blob = None
+                    try:
+                        blob = json.load(open('/'.join([root, file_name])))
+                    except Exception as e: # TODO find what kind of exceptions happen and why and handle
```

Comment:
eek, certainly not a fan of silently passing on all exceptions, i suggest you either narrow down or at least log something.

Reply:
yes - sorry, I'd be happy to change that into a "raise e" which is effectively what it was before, I put this here while trying to debug the failing attempt to open a file at this line.